laravel/routes/web.php

```diff
@@ -31,5 +31,10 @@
 Auth::routes();
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
 
+//プロフィール機能
+Route::resource('profile',UserController::class)->only([
```

Comment:
こちらはRESTful APIの設計に則った方が良いので、usersを使うようにしましょう。

ちなみにRESTful APIとは簡単に言うと、どのリソースに対してどのような処理を行うかをURLとHTTP METHODで分かるようにすることが目的となっています。

詳しくは調べてみてください🐱
```suggestion
Route::resource('users',UserController::class)->only([
```